resources/assets/js/validations.js

```diff
@@ -159,246 +144,231 @@ $.extend(true, laravelValidation, {
-        }, $.validator.format("The field must be beetwen {0} and {1} digits."));
+        },
 
         /**
          * Validate the size of an attribute.
+         * @return {boolean}
          */
-        $.validator.addMethod("laravelSize", function(value, element, params) {
-            return this.optional(element) ||
-                helpers.getSize(this, element,value) == params[0];
-        }, $.validator.format("The field must be {0}"));
+        Size: function(value, element, params) {
+            return laravelValidation.helpers.getSize(this, element,value) == params[0];
+        },
 
         /**
          * Validate the size of an attribute is between a set of values.
+         * @return {boolean}
          */
-        $.validator.addMethod("laravelBetween", function(value, element, params) {
-            return this.optional(element) ||
-                ( helpers.getSize(this, element,value) >= parseFloat(params[0]) && helpers.getSize(this,element,value) <= parseFloat(params[1]));
-        }, $.validator.format("The field must be between {0} and {1}"));
+        Between: function(value, element, params) {
+            return ( laravelValidation.helpers.getSize(this, element,value) >= parseFloat(params[0]) && laravelValidation.helpers.getSize(this,element,value) <= parseFloat(params[1]));
+        },
 
         /**
          * Validate the size of an attribute is greater than a minimum value.
+         * @return {boolean}
          */
-        $.validator.addMethod("laravelMin", function(value, element, params) {
-            return this.optional(element) ||
-                helpers.getSize(this, element,value) >= parseFloat(params[0]);
-        }, $.validator.format("The field must be at least {0}"));
+        Min: function(value, element, params) {
+            return laravelValidation.helpers.getSize(this, element,value) >= parseFloat(params[0]);
+        },
 
         /**
          * Validate the size of an attribute is less than a maximum value.
+         * @return {boolean}
          */
-        $.validator.addMethod("laravelMax", function(value, element, params) {
-            return this.optional(element) ||
-                helpers.getSize(this, element,value) <= parseFloat(params[0]);
-        }, $.validator.format("The field may not be greater than {0}"));
+        Max: function(value, element, params) {
+            return laravelValidation.helpers.getSize(this, element,value) <= parseFloat(params[0]);
+        },
 
         /**
          *  Validate an attribute is contained within a list of values.
+         * @return {boolean}
          */
-        $.validator.addMethod("laravelIn", function(value, element, params) {
-            return this.optional(element) ||
-                params.indexOf(value.toString()) != -1;
-        }, $.validator.format("The selected :attribute is invalid"));
+        In: function(value, element, params) {
+            return params.indexOf(value.toString()) != -1;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/albert-moreno-t/laravel-jsvalidation/file/842620834/issues/source?bid=2225045&fileBranchId=2805859#l279)